fanficfare/adapters/adapter_cfaa.py

```diff
@@ -0,0 +1,38 @@
+#  -*- coding: utf-8 -*-
+
+# Copyright 2020 FanFicFare team
```

Comment:
Copyright year should be set to current year for new files.

Reply:
noted, will update